pkgs/development/python-modules/pydeck/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, buildPythonApplication, fetchPypi, pkgs }:
+
+buildPythonApplication rec {
+  pname = "pydeck";
+  version = "0.7.0";
+
+  src = pkgs.python3Packages.fetchPypi {
```

Comment:
You don't need to qualify this with `pkgs.python3Packages.` if you're taking it as a direct argument above :point_up:

Reply:
fixed.